embeddable-say-editor/app/components/simple-editor.js

```diff
@@ -537,4 +553,19 @@ export default class SimpleEditorComponent extends Component {
     const { marks } = setup;
     marks.redacted = redacted;
   }
+
+  setupRdfaEditor(setup) {
+    const { plugins, uiConfig } = setup;
+
+    plugins.push(editableNodePlugin());
+    uiConfig.showRdfaEditor = true;
```

Comment:
Since it requires tools in the sidebar, this setup function should also make sure `uiConfig.sidebar = true`. Also, see other comment, I don't think `uiConfig.showRdfaEditor` is necessary.